src/FormFeedback.js

```diff
@@ -21,13 +22,17 @@ const FormFeedback = (props) => {
     className,
     cssModule,
     valid,
+    tooltip,
     tag: Tag,
     ...attributes
   } = props;
 
+  const validMode = tooltip ? 'tooltip' : 'feedback'
+
   const classes = mapToCssModules(classNames(
     className,
-    valid ? 'valid-feedback' : 'invalid-feedback'
+    'd-block',
```

Comment:
this class needs to be removed. It's causing the feedbacks to display when the shouldn't.

Reply:
Oks, i was trying this fast approach to solve an issue with the class `valid-tooltip` that has `display: none`
In the bootstrap examples, the `display:block` is applied to this elements through some parent classes like `was-validated` and some other structures → https://imgur.com/a/Ds5z50D

I don't know how to replicate this behaviour here.
Any suggestions?

Reply:
Something like this could be a good solution for now?

```
const validMode = tooltip ? 'tooltip' : 'feedback';
const classes = mapToCssModules(
    classNames(
      className,
      (validMode === 'tooltip') && 'd-block',
      valid ? `valid-${validMode}` : `invalid-${validMode}`
    ),
    cssModule
  );
```

Reply:
Forget it!! I wasn't get it how this is working. I think it's ok now